cli/alsatian-cli.ts

```diff
@@ -1,4 +1,6 @@
 #! /usr/bin/env node
+// tslint:disable-next-line
+require('ts-node/register');
```

Comment:
Could you change this to

```typescript
import "tsnode/register";
```

I think this should work and satisfy tslint too

Reply:
Done :-)